swagger.yaml

```diff
@@ -3671,6 +3671,9 @@ definitions:
         x-go-custom-tag: gorm:"type:text"
         type: string
         description: Array of image statuses.
+      domain_name_resolution:
```

Comment:
I guess that when we'll want to add more domain validations in the future, than we might want to have a different "Validation" for each one, some maybe rename this to `api_domain_name_resolution`

Reply:
Will each of the resolutions will have each own step-type and DB field ?

Reply:
Seems like this conversation was accidentally resolved.

@eranco74 already opened a [ticket](https://issues.redhat.com/browse/MGMT-6911) for another domain validation we may want to do.

>Will each of the resolutions will have each own step-type and DB field ?

No, just one step and one DB column - the resolution request may contain multiple domains and will always sent to the agent containing all the domains we may ever want to resolve, then the response will be saved as a JSON in the database.

However, we may want separate validations with different messages for each of those domains.

> Will it also have conditions on when to run it (none platform only - for example) ?

Not sure about that one, maybe @eranco74 can say about his new domain resolution ticket in what platforms it's required